src/code/z_message.c

```diff
@@ -356,9 +358,75 @@ void Message_GrowTextbox(MessageContext* msgCtx) {
     R_TEXTBOX_X = (R_TEXTBOX_X_TARGET + R_TEXTBOX_WIDTH_TARGET) - (R_TEXTBOX_WIDTH / 2);
 }
 
-#if OOT_PAL // TODO: implement NTSC version
+#if OOT_NTSC
+
+void Message_FindMessageJPN(PlayState* play, u16 textId) {
+    const char* foundSeg;
+    const char* nextSeg;
+    const char* seg;
+    Font* font = &play->msgCtx.font;
+    MessageTableEntry* messageTableEntry = sJpnMessageEntryTablePtr;
+
+    seg = messageTableEntry->segment;
+
+    while (messageTableEntry->textId != 0xFFFF) {
+        if (messageTableEntry->textId == textId) {
+            foundSeg = messageTableEntry->segment;
+            font->charTexBuf[0] = messageTableEntry->typePos;
+            messageTableEntry++;
+            nextSeg = messageTableEntry->segment;
+            font->msgOffset = foundSeg - seg;
+            font->msgLength = nextSeg - foundSeg;
+            return;
+        }
+        messageTableEntry++;
+    }
+
+    messageTableEntry = sJpnMessageEntryTablePtr;
+
+    foundSeg = messageTableEntry->segment;
+    font->charTexBuf[0] = messageTableEntry->typePos;
+    messageTableEntry++;
+    nextSeg = messageTableEntry->segment;
+
+    font->msgOffset = foundSeg - seg;
+    font->msgLength = nextSeg - foundSeg;
+}
 
-void Message_FindMessage(PlayState* play, u16 textId) {
+void Message_FindMessageNES(PlayState* play, u16 textId) {
+    const char* foundSeg;
```

Comment:
never was a fan of NES, what does it even mean? I know its an acronym they originally used but it never made sense to me

Reply:
I actually think the name fits pretty well here because it looks up the message in `sNesMessageEntryTable`. I don't think it made sense in `Message_DrawTextNES` though because it also handled German and French text. I renamed these to `Message_DrawText` and `Message_DrawTextWide`, using the wide/non-wide naming in message_data_fmt.h. What do you think?

Reply:
My point is moreso that NES as an acronym is bad as a whole, and shouldn't be used at all. So yeah its true it does match `sNesMessageEntryTable`, but I think that should probably change too.

That can happen separately from this PR, but I think it should be done soon before we propagate this bad acronym in lots of other places in the codebase